controllers/datafoundation.go

```diff
@@ -243,6 +258,20 @@ func (r *dataFoundationReconciler) parseSpec(offering *v1alpha1.ManagedFusionOff
 	return nil
 }
 
+func (r *dataFoundationReconciler) reconcileWatchers() error {
+	if !r.ManagedFusionOfferingReconciler.AvailableCRDs[StorageClusterCRD] {
+		crd := apiextensionsv1.CustomResourceDefinition{}
+		crd.Name = StorageClusterCRD
+		// we only check for storagecluster CRD because other crds are installed as a dependency via ocs-operator
+		if err := r.get(&crd); err != nil {
+			return err
+		} else {
+			utils.HandleMissingWatchers()
```

Comment:
How does the util function know which watch is missing? you are not passing any kind of information to the util func